backend/internal/data/sql/queries/shutter_explorer.sql

```diff
@@ -119,3 +119,37 @@ FROM (
 ) AS latest_events
 WHERE 
     is_registeration = TRUE;
+
+-- name: QuerySlotAndValidatorDataByEpoch :many
+SELECT 
+    pd.public_key, 
+    pd.validator_index, 
+    pd.slot,
+	pd.epoch,
```

Comment:
indentation typo?

Reply:
i think it was github code viewer issue, but we have gotten rid of epoch anyways now